hnn_core/network.py

```diff
@@ -327,11 +332,12 @@ class Network(object):
     ----
     `net = jones_2009_model(params)` is the reccomended path for creating a
     network. Instantiating the network as `net = Network(params)` will
-    produce a network with no cell to cell connections. As such,
+    produce a network with no cell-to-cell connections. As such,
     connectivity information contained in `params` will be ignored.
     """
 
     def __init__(self, params, add_drives_from_params=False,
+                 inplane_distance=1., layer_separation=1307.4,
```

Comment:
these two parameters are not documented in public class

Reply:
These were removed, more discussion is needed.